include/pulsar/c/message.h

```diff
@@ -33,6 +33,15 @@ typedef struct _pulsar_message pulsar_message_t;
 typedef struct _pulsar_message_id pulsar_message_id_t;
 
 PULSAR_PUBLIC pulsar_message_t *pulsar_message_create();
+/**
+ * Copy the contents of one pulsar_message_t object to another.
+ *
+ * Note: This method is a shallow copy, which will copy the pulsar::Message.
+ *
+ * @param from  A pointer to the pulsar_message_t object that you want to copy from
+ * @param to  A pointer to the pulsar_message_t object that you want to copy to
+ */
+PULSAR_PUBLIC void pulsar_message_copy(pulsar_message_t *from, pulsar_message_t *to);
```

Comment:
```suggestion
PULSAR_PUBLIC void pulsar_message_copy(const pulsar_message_t *from, pulsar_message_t *to);
```

Use `const` to indicate the object that `from` points to won't change